_posts/2023-01-success-digital-signage.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title: "Success Story: WPE in digital signage"
+date: 2023-01-06
+tags: [blogpost, success]
+permalink: /blog/2023-success-digital-signage.html
+preview: Digital signage web rendering players have many advantages and are a trend nowadays.
+thumbnail: /assets/img/logo-digital-signage@2x.png
+---
+
+<div class="success-top">
+<img alt="WPE WebKit in digital signage" align="center" src="{{ '/assets/img/logo-digital-signage@2x.png' | url }}">
+<img alt="WPE" align="center" src="{{ '/assets/img/logo-blue.svg' | url }}">
```

Comment:
The `|url` filter does _not_ need to be used anymore since the update to Eleventy 2.0 done in  #160. This can be simplified as follows:

```suggestion
<img alt="WPE WebKit in digital signage" align="center" src="/assets/img/logo-digital-signage@2x.png">
<img alt="WPE" align="center" src="/assets/img/logo-blue.svg">
```

Reply:
Done.